test/unit/specs/components/search-grid-filter.spec.js

```diff
@@ -109,7 +110,7 @@ describe('SearchGridFilter', () => {
     })
     expect(checkedFilters.length).toEqual(0)
     // Filters are reset with the initial `filterData`
-    expect(uncheckedFilters.length).toEqual(22)
+    expect(uncheckedFilters.length).toEqual(25)
```

Comment:
Why is this changing?

Reply:
Because previously we didn't have the `sizes` in mock filters in this test. Without them, the filter handling in search store fails. And after adding the `sizes` option, the number of filters goes up 3 (for `small`, `medium` and `large`, IIRC). The filters `filtering` (or typing, maybe) needs to be a bit more robust, I guess, because sometimes tests pass non-array objects to some functions that use `.filter` over them.

Reply:
Ah okay, thanks for explaining 🙂 I saw that we added the sizes to the filter but I didn't realize it added three more options instead of just one size 😂 But I get it now!